package.json

```diff
@@ -49,6 +49,7 @@
     "qs": "^6.9.4",
     "re-resizable": "^6.5.4",
     "react": "^16.13.1",
+    "react-color": "2.19.3",
```

Comment:
This lib looks pretty big.. how did we choose it? And are there better alternatives?

- https://bundlephobia.com/package/react-color@2.19.3
  - 143.1kB minified
  - 36.3kB minified+gzipped

Reply:
@0xdevalias If the size is too big we can look for an alternative, as long as product likes the visual part of it.

Reply:
Doesn't look like they're too attentive to sizing given they seem to include both `lodash` and `lodash-es` too..

![image](https://user-images.githubusercontent.com/753891/124943272-b219b700-e04f-11eb-8ace-f03c92f9db60.png)

Reply:
Would be good to do a build/bundle analyze after this to see how much it ends up as, as I know some of my recent optimisations will automagically cut out both of those lodash's for example, and it's apparently tree-shakeable too.. so might end up much smaller in our bundle.